ensembl_dbcopy/migrations/0003_auto_20201119_1644.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 2.2.13 on 2020-11-19 16:44
```

Comment:
Can you rename the automated generated name of file to something more meaningful? And I am not sure all of these are required for the updates you wrote. e.g.: 
```
migrations.AlterField(
            model_name='group',
            name='host_id',
            field=models.ForeignKey(db_column='auto_id', on_delete=django.db.models.deletion.CASCADE, related_name='groups', to='ensembl_dbcopy.Host'),
        ),
```

Reply:
I'll rename the migration, although there's more than a field addition. I have to say, I struggled more that I should have with these migrations. I tried with only the `AddField()` which seems truly the only difference since I've only added one field to the model but without success (the `migrate` command failed and the ORM wasn't recognizing the field without the migration, very weird). So I've simply ran `makemigrations` and then `migrate` and everything magically worked fine. Same happened on our test server.

Reply:
Tace care that the migrate actually do something like that. Since we are using multiple databases with only one migration table. this can lead to errors. The nicer solution would be to have a "django_migrations" table inside `ensembl_dbcopy` database.